waku/waku_archive/retention_policy/retention_policy_size.nim

```diff
@@ -42,21 +42,12 @@ method execute*(p: SizeRetentionPolicy,
                 driver: ArchiveDriver):
                 Future[RetentionPolicyResult[void]] {.async.} =
   ## when db size overshoots the database limit, shread 20% of outdated messages 
-
-  # get page size of database
-  let pageSizeRes = await driver.getPagesSize()
-  let pageSize: int64 = int64(pageSizeRes.valueOr(0) div 1024)
-
-  if pageSize == 0:
-    return err("failed to get Page size: " & pageSizeRes.error)
-
-  # to get the size of the database, pageCount and PageSize is required
-  # get page count in "messages" database
-  let pageCount = (await driver.getPagesCount()).valueOr:
-    return err("failed to get Pages count: " & $error)
+  # get size of database
+  let dbSize = (await driver.getDatabasesSize()).valueOr:
+    return err("failed to get database size: " & $error)
 
   # database size in megabytes (Mb)
```

Comment:
I think this is because it makes it more readable, when user inputs the size based retention policy it uses "MB" "GB" but that's because it is more readable. There are many instances in our code where MB is associated with 1024 type notation.

Reply:
Yes `MiB` is the correct one

Reply:
I'll add an issue for the whole nwaku to check MB vs MiB

Reply:
Ok perfect! I vote for following the discussion in the #2247 as it is beyond the scope of this PR.

Reply:
I have now changed the database unit of processing to bytes to clear up the confusion, in later PR I will resolve the user facing config file/parameter input unit with due conversation.